### ISSUE TITLE
vue-infinity-scroll

### ISSUE THREAD

**Issue by auvipy** [vue-component/request#1]

**Comment by djyde** [comment]
You mean when scroll to bottom, the page will auto load data?

**Comment by auvipy** [comment]
yes. you could check ng-inifinity-scroll/jquery infinity scroll :)

**Comment by djyde** [comment]
got it, good idea

**Comment by egoist** [comment]
Hmm, might be years late, but I did build one 😅 https://github.com/egoist/vue-mugen-scroll
